cinema/permissions.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
There is a syntax error in the import statement. You should use a comma to separate the imported modules: `from rest_framework.permissions import BasePermission, SAFE_METHODS`.